data/hai/hai reveal 2 start.txt

```diff
@@ -10,10 +10,34 @@
 
 ########################################################################
 
-#Investigate a theorised smuggling route and rescue Hai. You have to do at least three of these missions.
+#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.
 
 #NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.
 
+mission "Hai Rescue: Yeertle Hint"
+	landing
+	name "Investigate a Were-Squirrel"
+	description "A truly outlandish rumor has been floating around that might be a Hai."
```

Comment:
```suggestion
	description "A truly outlandish rumor has been floating around that might be a Hai hiding out on <destination>."
```
The map will show the destination anyways, so I don't see any reason not to include it in the description (since that seems to be rule of thumb).